thesis-de/content-de.tex

```diff
@@ -2583,7 +2584,7 @@ \section{Realisierung Grafana}
 Graphen angepasst, damit der Graph Werte visualisieren kann.  Anbei ein
 Beispiel, für eine solche Abfrage:
```

Comment:
der satzaufbau passt hier nicht. man erwartet, das ein bild kommt. da kommt aber keins. da kommt eine verlinkung in den anhang. entweder binde das bild direkt ein oder bau den satz um